WcaOnRails/db/structure.sql

```diff
@@ -54,11 +54,15 @@ CREATE TABLE `Competitions` (
   `announced_at` datetime DEFAULT NULL,
   `base_entry_fee_lowest_denomination` int(11) DEFAULT NULL,
   `currency_code` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
-  `connected_stripe_account_id` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
```

Comment:
Looks like it just moved down a couple of lines for some reason? I don't know what that's all about. I went through and cleaned up the rest of the diff in this file (which turned up #1325)